pyproject.toml

```
@@ -38,6 +38,23 @@ classifiers = [
     "Topic :: Utilities",
 ]
 
+dependencies = [
+    "importlib-resources >= 6.0",
```

This is wrong, we only require importlib-resources package on older Python versions.

```suggestion
"importlib-resources >= 6.0; python_version < '3.10'"
```

---

Restoring